homeassistant/components/owntracks/__init__.py

```diff
@@ -180,7 +181,33 @@
             return json_response([])
 
     hass.helpers.dispatcher.async_dispatcher_send(DOMAIN, hass, context, message)
-    return json_response([])
+
+    response = []
+
+    persons = list(filter(lambda x: x.domain == "person", hass.states.async_all()))
+
+    for person in persons:
+        response.append(
+            {
+                "_type": "location",
+                "lat": person.attributes["latitude"],
+                "lon": person.attributes["longitude"],
+                "tid": person.attributes["id"][0:2],
```

Comment:
Why are you cutting off a part of the ID. That's not how IDs work…

Reply:
The "tid" in Owntracks is just 2 characters, and is generated in a similar way - the last 2 characters of the device's name - https://owntracks.org/booklet/features/tid/#tid